.github/workflows/ci.yml

```diff
@@ -43,7 +43,21 @@ jobs:
     - name: Build PDF
       if: github.repository == 'tldr-pages/tldr' && github.ref == 'refs/heads/main'
       working-directory: ./scripts/pdf
-      run: python render.py ../../pages -c solarized-light
+      run: |
+        for path in ../../pages*; do
+          folder=$(basename "${path}")
+          language="${folder##*.}"
+          case $folder in
+            pages.bn | pages.ja | pages.ko | pages.ml | pages.ta | pages.th | pages.zh | pages.zh_TW)
+              ;;
+            pages)
+              python render.py ${path} -c solarized-light
+              ;;
+            *)
+              python render.py ${path} -c basic -o tldr-pages-${language}.pdf
+              ;;
+          esac
+        done
```

Comment:
Agreed, also I think we can continue to use the `tldr-book.pdf` naming (will update the relevant places to reflect it). I think for the English page alone I will rename the document during deploy (so that existing places [i.e search engine listings, etc] that are referring to it aren't affected).